install/linux/docker-ee/ubuntu.md

```diff
@@ -247,13 +247,20 @@ To upgrade Docker Engine - Enterprise, follow the steps below:
 ### Install from a package
 
 If you cannot use Docker's repository to install Docker Engine - Enterprise, you
-can download the `.deb` file for your release and install it manually. You need
-to download a new file each time you want to upgrade Docker Engine - Enterprise.
+can download the `.deb` files for your release and install them manually. You
+need to download a new file or set of files each time you want to upgrade Docker
+Engine - Enterprise.
 
 1.  Go to the Docker Engine - Enterprise repository URL associated with your
     trial or subscription in your browser. Go to
-    `ubuntu/x86_64/stable-<VERSION>` and download the `.deb` file for the
-    Docker Engine - Enterprise version and architecture you want to install.
+    `ubuntu/dists/<distribution>/pool/stable-<VERSION>` and download the `.deb`
+    file(s) for the Docker Engine - Enterprise version and architecture you want
+    to install.
+
+    > **Note:**
+    > For newer versions of Docker, you will download three files. They
+    > are `docker-ee-cli_<version>.deb`, `containerd.io_<version>.deb`, and
+    > `docker-ee_<version>.deb`.
```

Comment:
Can we specify which version onward?

Reply:
These are the latest for the respective component.

2019-10-18T02:59:30Z | 20027634 | containerd.io_1.2.10-3_amd64.deb

2019-11-14T19:07:30Z | 101666388 | docker-ee-cli_19.03.5~3-0~ubuntu-bionic_amd64.deb

2019-11-14T19:07:30Z | 24839502 | docker-ee_19.03.5~3-0~ubuntu-bionic_amd64.deb